libs/awscdk/src/bucket.ts

```diff
@@ -230,6 +231,14 @@ export class Bucket extends cloud.Bucket {
   private envName(): string {
     return `BUCKET_NAME_${this.node.addr.slice(-8)}`;
   }
+
+  public get arn(): string {
```

Comment:
Why are these needed?

Reply:
I didn't understand, Elad.

Do you want me not to expose the bucket arn?

Because I find it interesting to expose it.

Eventually, someone might want to create a library that uses Event Bridge or Step Function and wishes to connect directly to a bucket. Exposing the ARN would make it easy to do that

Reply:
My mistake. Ignore my comment. I didn't realize it's a property...